flocker/control/test/test_persistence.py

```diff
@@ -479,7 +431,7 @@ def _build_node(applications):
     APPLICATIONS,
     # If we add this hint on the number of applications, Hypothesis is able to
     # run many more tests.
-    average_size=10,
+    average_size=3,
```

Comment:
Related, from `#hypothesis` earlier today:

```
[15:30:13]  <DRMacIver>	jml: BTW it looks like you can speed your tests up quite a lot if you just replace your usage of text() with text(average_size=5)
```

I would suggest giving this idiom a name: e.g. `short_text`, or `label`.

(UK time)